components/eamxx/src/share/io/scream_io_control.hpp

```diff
@@ -39,6 +39,30 @@ struct IOControl {
                                         ts.get_time()==next_write_ts.get_time());
   }
 
+  void set_frequency_units (const std::string& freq_unit) {
+    if (freq_unit=="none" or freq_unit=="never") {
+      frequency_units = freq_unit;
+    } else if (freq_unit=="nstep" or freq_unit=="nsteps") {
+      frequency_units = "nsteps";
+    } else if (freq_unit=="nsecond" or freq_unit=="nseconds" or freq_unit=="nsecs") {
```

Comment:
Do we have to support also "nsec" and "nmin"?

Reply:
oh wait, I misunderstood the question.  Right, nsec"s" and nmin"s" are from EAMxx so EAM won't ever set them or `nsec` or `nmin`